### PR TITLE
config: updated for nginx 1.9.11+

### PR DESCRIPTION
**Pull request by doujiang24**

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -30,6 +30,13 @@ ECHO_DEPS="                                                                 \
         $ngx_addon_dir/src/ngx_http_echo_foreach.h                          \
         "
 
+# nginx won't have HTTP_POSTPONE_FILTER_MODULE & HTTP_POSTPONE_FILTER_SRCS
+# defined since 1.9.11
+if test -z "$HTTP_POSTPONE_FILTER_MODULE"; then
+    HTTP_POSTPONE_FILTER_MODULE=ngx_http_postpone_filter_module
+    HTTP_POSTPONE_FILTER_SRCS=src/http/ngx_http_postpone_filter_module.c
+fi
```
Comment (agentzh) [inline config]: Do you think it's better to replace `fi` here with `else`? I think it's better to skip the following code snippet altogether for nginx 1.9.11+. What do you think?

Comment (doujiang24) [inline config]: I think we can't skip them. We also need them even in nginx 1.9.11+.

Comment (agentzh) [inline config]: @doujiang24 Oh I didn't know. Thanks for the info!

Comment (anthonyryan1) [inline config]: Just to chime in on this with some confirmation. Skipping those will break building this `--without-http_ssi_module` or something else to pull in HTTP_POSTPONE.

I feel like what we really need is a better way to tell Nginx we depend on that and it needs to be included.

**Comment by anthonyryan1** [comment]
I sent an email to the nginx-devel mailing list, and the [response](http://mailman.nginx.org/pipermail/nginx-devel/2016-April/008172.html) makes it pretty clear the upstream doesn't care about this problem. They feel it's good enough to simply tell users they need to include at least one of the built-in modules which pulls in postpone.

I personally feel we should throw in the towel on this one, just document the limitation properly (that we depend on postpone, and there's no direct way to enable it) and how to work around it. I'll prepare a separate pull request to do so in a few minutes.

**Comment by agentzh** [comment]
Merged into master. Thanks

**Pull request closed without merging**